src/pages/courseSettings.js

```diff
@@ -0,0 +1,269 @@
+                <AddStaff onAddStaff={staff => this.addStaff(staff)} />
+                <FlipMove
+                  enterAnimation="accordionVertical"
+                  leaveAnimation="accordionVertical"
+                  duration={200}
+                  typeName={null}
+                >
+                  {users}
+                </FlipMove>
+              </ListGroup>
+            </Card>
+          </Container>
+        </div>
+        <ShowForAdmin>
+          <div className="d-flex flex-wrap align-items-center mb-4">
+            <Container>
+              <Card className="staff-card">
+                <CardHeader className="bg-primary text-white d-flex align-items-center">
+                  <CardTitle tag="h4" className="mb-0">
+                    Change {this.props.course.name}'s Listing
+                  </CardTitle>
+                </CardHeader>
+                <Form>
+                  <Col sm={9}>
+                    <CustomInput
+                      id="isUnlisted"
+                      type="switch"
+                      name="isUnlisted"
+                      defaultChecked={this.props.course.isUnlisted}
+                      onChange={e =>
+                        this.updateUnlisted({ isUnlisted: e.target.checked })
+                      }
+                    />
+                    <FormText color="muted">
+                      Making your course unlisted will only allow students with
+                      the course shortcode to view this course.
+                    </FormText>
+                  </Col>
+                </Form>
+              </Card>
+            </Container>
+          </div>
+        </ShowForAdmin>
+
+        <div className="d-flex flex-wrap align-items-center mb-4">
+          <Container>
+            <Card className="staff-card">
+              <CardHeader className="bg-primary text-white d-flex align-items-center">
+                <CardTitle tag="h4" className="mb-0">
+                  Show Question Feedback Form
+                </CardTitle>
+              </CardHeader>
+              <Form>
+                <Col sm={9}>
+                  <CustomInput
+                    id="questionFeedback"
+                    type="switch"
+                    name="questionFeedback"
+                    defaultChecked={this.props.course.questionFeedback}
+                    onChange={e =>
+                      this.updateQuestionFeedback({
+                        questionFeedback: e.target.checked,
+                      })
+                    }
+                  />
+                  <FormText color="muted">
+                    Allowing question feedback will let your course staff
+                    provide feedback after answering each student's question.
+                  </FormText>
+                </Col>
+              </Form>
+            </Card>
+          </Container>
+        </div>
+
+        <div className="d-flex flex-wrap align-items-center mb-4">
```

Comment:
Instead of a lone button, could we aim for something more like PrairieLearn's downloads section? Notably, text descriptions and the specific filename. This will allow us to easily add other types of downloads in the future, and seeing a text description of what the data contains would be awesome.
<img width="1122" alt="Screen Shot 2019-11-02 at 4 45 40 AM" src="https://user-images.githubusercontent.com/1476544/68070452-b5fa1f80-fd2b-11e9-8372-57de467b4e0e.png">